scripts/visualization/reflex_mapa_licea.py

```
@@ -0,0 +1,171 @@
+import reflex as rx
+import pandas as pd
```

Ruff (0.11.9)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)